inventory.py

```diff
@@ -1,14 +1,17 @@
 # standard libraries
 import json, logging, os
-from datetime import datetime
 from json.decoder import JSONDecodeError
 from typing import Dict, Sequence, Union
+import time
 
 # third-party libraries
 import pandas as pd
 import psycopg2
 from requests import Response
 from umich_api.api_utils import ApiUtil
+from published_date.published_date import FetchPublishedDate
```

Comment:
Can you put this in under local libraries with `db_creator`?

Reply:
I don't think that makes sense. Since this is simply pulling data and no DB transactions

Reply:
It's a local library though, not a third-party one?

Reply:
Yes, It is probably ok For now. We can have as many packages are we want. If we have to make a package we can make it as `canvas` and have different canvas data under this package like published date, user, enrollment, list of published courses.

Reply:
I'm just trying to keep this organized. I find it useful to separate imports that are for libraries we maintain in this project and those we don't.

Reply:
I renamed the package name to the canvas. we can have all the canvas related data in that package

Reply:
This is not a bad idea, but it wasn't what I was asking you to do. I was just asking you to move your import down under the "# local libraries" comment.

Reply:
Thank you!